perf/bin/tachyon-perf-log-collect

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+function printUsage {
+  echo "Usage: tachyon-perf-log-collect <all|[Nodes ...]>"
+  echo "  'all' means collect the logs from all the slave nodes"
+  echo "  or you can specify the nodes such as 'node1 node2 ...'"
+}
+
+# if less than 1 args specified, show usage
+if [ $# -le 0 ]; then
+  printUsage
+  exit 1
+fi
+
+bin=`cd "$( dirname "$0" )"; pwd`
+
+DEFAULT_PERF_LIBEXEC_DIR="$bin"/../libexec
+TACHYON_PERF_LIBEXEC_DIR=${TACHYON_PERF_LIBEXEC_DIR:-$DEFAULT_PERF_LIBEXEC_DIR}
+. $TACHYON_PERF_LIBEXEC_DIR/tachyon-perf-config.sh
+
+TACHYON_PERF_LOG_DIR=$TACHYON_PERF_HOME/logs
+TACHYON_PERF_COLLECT_LOG_DIR=$TACHYON_PERF_LOG_DIR/collected-logs
+
+# collect logs from all the slave nodes
+if [ "$1"x = "all"x ]; then
```

Comment:
In case that the compared string is empty, but here seems $1 won't be empty. I'll remove the `x`.